Content.Server/Mind/MindSystem.cs

```diff
@@ -146,22 +143,24 @@ private void OnMindContainerTerminating(EntityUid uid, MindContainerComponent co
         if (_gameTicker.RunLevel == GameRunLevel.PreRoundLobby)
             return;
 
-        if (component.Mind is not { } mind)
+        if (!TryComp(component.Mind, out MindComponent? mind))
             return;
 
+        var mindId = component.Mind.Value;
+
         // If the player is currently visiting some other entity, simply attach to that entity.
         if (mind.VisitingEntity is {Valid: true} visiting
             && visiting != uid
             && !Deleted(visiting)
             && !Terminating(visiting))
         {
-            TransferTo(mind, visiting);
+            TransferTo(mindId, visiting);
             if (TryComp(visiting, out GhostComponent? ghost))
                 _ghostSystem.SetCanReturnToBody(ghost, false);
             return;
         }
 
-        TransferTo(mind, null, createGhost: false);
+        TransferTo(mindId, null, createGhost: false);
```

Comment:
I'll create a PR for this.

Reply:
Merged in https://github.com/DrSmugleaf/space-station-14/pull/30